doc/nrf/libraries/nrf_rpc/nrf_rpc_utils.rst

```diff
@@ -0,0 +1,17 @@
+.. _nrf_rpc_utils:
+
+nRF RPC utilities
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF RPC Utilities library allows you to use the :ref:`nrf_rpc` common useful functions on remote server.
```

Comment:
```suggestion
The nRF RPC Utilities library allows you to use the :ref:`nrf_rpc` common functions on remote server.
```
"useful" is a value judgement, and we try to keep that from the technical documentation.